db/migrate/20190618175458_create_votes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVotes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :votes do |t|
+      t.integer :user_id
+      t.integer :votable_id
+      t.string :votable_type
+      #True for upvote, False for downvote
+      t.boolean :vote_type
```

Comment:
This is great! 

Something to consider for a future refactor or just a thought experiment: If you made this an integer field with values of 1 or -1, you could use `sum` functions to calculate the overall score.